opcua-core/Cargo.toml

```
@@ -28,6 +28,7 @@ url = { workspace = true }
 
 opcua-types = { path = "../opcua-types" }
 opcua-crypto = { path = "../opcua-crypto" }
+thiserror.workspace = true
```

Didn't see this format being used before (didn't even know it was possible to be honest 😉) but maybe it's best to keep things consistent and write this as:
```suggestion
thiserror = { workspace = true )
```